src/modules/reporting/actions/load-reporting-window-bounds.js

```diff
@@ -13,9 +13,9 @@ export const loadReportingWindowBounds = (callback = logError) => (dispatch, get
       if (err) return callback(err)
 
       dispatch(updateReportingWindowStats({
-        startTime: result.startTime,
-        endTime: result.endTime,
-        stake: result.totalStake || 0,
+        startTime: (result || {}).startTime,
```

Comment:
Could this be rewritten: `(err, result={}) => {`?

Reply:
I don't think so, because `result` is an explicit `null` instead of not being passed at all.

Reply:
Good call:
![image](https://user-images.githubusercontent.com/328965/38524486-5a521cb6-3c14-11e8-8e84-0bb94042a29d.png)